### ISSUE TITLE
File Block: Consider removing the toggle control help text under the "Show Download Button" toggle.

### ISSUE THREAD

**Issue by kjellr** [WordPress/gutenberg#13429]
The "Show Download Button" toggle for the File block comes with some helper text to explain the current state: 

![screen shot 2019-01-22 at 12 48 22 pm](https://user-images.githubusercontent.com/1202812/51555310-55a58b80-1e45-11e9-927e-de833e6c33be.png)

There are many instances where supplementary text is helpful (For instance, we use that text to [explain what a drop cap is](https://cloudup.com/c7fBgamSZrk) for the paragraph block). 

In the case of the file block's download button however, this text is simply a readout of the toggle button's state. It seems redundant and we should consider removing it.

**Comment by afercia** [comment]
See the lengthy discussion on https://github.com/WordPress/gutenberg/issues/2146 which was closed without consensus.

From an accessibility perspective, the accessibility team asked to have a _textual representation_ of the toggle state. An On/Off was enough. Instead, in many cases the help text was used as sort of lengthier "dynamic" description.

Currently there's some inconsistency across all these toggles. I'd be in favor of removing inappropriate (slightly redundant) text but: graphics is not enough, there need to be some text to indicate the state.

**Comment by afercia** [comment]
I'd agree this specific wording is redundant but there should be an `On / Off` text or maybe `Enabled / Disabled`. I'd like to not repeat all the argumentations already made on #2146 🙂but I'd suggest to have a look at it.

**Comment by kjellr** [comment]
Thanks, @afercia. That `on/off` label sounds like something that should be tackled globally for all toggles. In the meantime, I've opened #13485 for this specific File Download button text.
